lib/Sources/BluetoothEngine/BluetoothEngine.swift

```diff
@@ -158,4 +165,19 @@ public actor BluetoothEngine: JsMessageProcessor {
         guard try predicate(currentState) == false else { return }
         _ = try await client.awaitSystemState(predicate: predicate)
     }
+
+    private func logRequest(message: Message) {
+        guard enableDebugLogging else { return }
+        messageLog.debug("Request \(message.action.rawValue, privacy: .public): \(JsType.dictionaryAsString(message.rawRequestData), privacy: .public)")
```

Comment:
The OSLog magic works differently to `print` and as such you cannot just dump an `Any` into the interpolation string. So we have to convert the Js anonymous data into strings here (and below).